src/index.js

```diff
@@ -11,6 +11,7 @@
  */
 
 export * as contextHelper from './context.js';
+export { ContextHelper } from './context.js';
```

Comment:
why are there two imports for `context.js`?

Reply:
Let me try moving those into one, I kept the old export so it would be backwards compatible and marked all the old methods as deprecated so we can upgrade this without immediately refactoring a bunch of code (esp tests)

Reply:
:/ that is fine. Just add a comment that one is for backwards compatibility